clients/dotnet/WebClient/MemoryWebClient.cs

```diff
@@ -299,6 +300,52 @@ public async Task DeleteDocumentAsync(string documentId, string? index = null, C
         return JsonSerializer.Deserialize<MemoryAnswer>(json, s_caseInsensitiveJsonOptions) ?? new MemoryAnswer();
     }
 
+    /// <inheritdoc />
+    public async IAsyncEnumerable<string> AskStreamingAsync(
+        string question,
+        string? index = null,
+        MemoryFilter? filter = null,
+        ICollection<MemoryFilter>? filters = null,
+        double minRelevance = 0,
+        [EnumeratorCancellation] CancellationToken cancellationToken = default)
+    {
+        if (filter != null)
+        {
+            if (filters == null) { filters = new List<MemoryFilter>(); }
+
+            filters.Add(filter);
+        }
+
+        MemoryQuery request = new()
+        {
+            Index = index,
+            Question = question,
+            Filters = (filters is { Count: > 0 }) ? filters.ToList() : new(),
+            MinRelevance = minRelevance
+        };
+        using StringContent content = new(JsonSerializer.Serialize(request), Encoding.UTF8, "application/json");
+
+        using var httpRequest = new HttpRequestMessage(HttpMethod.Post, Constants.HttpAskStreamEndpoint);
+        httpRequest.Content = content;
+
+        HttpResponseMessage response = await this._client.SendAsync(httpRequest, cancellationToken).ConfigureAwait(false);
```

Comment:
This call buffers the whole response for some reason. I can't figure out why.
- [ ]  Fix buffering

Reply:
Thanks! @marcominerva. I was able to fix it with those changes. 

The response in the webclient when printed is (these are the exact characters, not a stringified list):
```json
["Yes",","," NASA"," has"," invited"," media"," to"," see"," the"," new"," test"," version"," of"," the"," Orion"," spacecraft"," and"," the"," hardware"," teams"," will"," use"," to"," recover"," the"," capsule"," and"," astronauts"," upon"," their"," return"," from"," space"," during"," the"," Artem","is"," II"," mission","."," The"," event"," will"," take"," place"," at"," Naval"," Base"," San"," Diego"," on"," August"," ","2","."," Personnel"," involved"," in"," recovery"," operations"," from"," NASA",","," the"," U",".S","."," Navy",","," and"," the"," U",".S","."," Air"," Force"," will"," be"," available"," to"," speak"," with"," media","."," Teams"," are"," currently"," conducting"," tests"," in"," the"," Pacific"," Ocean"," to"," demonstrate"," and"," evaluate"," the"," processes",","," procedures",","," and"," hardware"," for"," recovery"," operations"," for"," crew","ed"," Artem","is"," missions","."," The"," tests"," will"," help"," prepare"," the"," team"," for"," Artem","is"," II",","," NASA","'s"," first"," crew","ed"," mission"," under"," Artem","is"," that"," will"," send"," four"," astronauts"," in"," Orion"," around"," the"," Moon"," to"," checkout"," systems"," ahead"," of"," future"," lunar"," missions","."," The"," Artem","is"," II"," crew"," will"," participate"," in"," recovery"," testing"," at"," sea"," next"," year","."]
```
Which is not very clean, and is not really usable.

If i swap the whole codeblock for .NET8.0's ReadFromJsonAsAsyncEnumerable<>() method it works and give a clean string as response, but that would mean migrating the WebClient project to .NET8.0.

Reply:
The `ReadFromJsonAsAsyncEnumerable<>()` method should be available in the https://www.nuget.org/packages/System.Net.Http.Json NuGet package, that is compatible also for .NET Standard 2.0. You can try adding this package to the project.

Reply:
Great point! Added the dependency and it works!
The output in the webclient example prints as it is supposed to now.